### ISSUE TITLE
Refactor HiveTestUtils#SESSION to class Session

### ISSUE THREAD

**Issue by shixuan-fan** [prestodb/presto#12367]
We should make `HiveTestUtils#SESSION` (https://github.com/prestodb/presto/blob/43bd519052ba4c56ff1f4fc807075637ab5f4f10/presto-hive/src/test/java/com/facebook/presto/hive/HiveTestUtils.java#L57)  a `Session` class instead of a `ConnectorSession`, and use `toConnectorSession` for current usage.

This would help us reduce code modification when default `SESSION` changes and make test more consistent. A follow-up would be address the TODO in #12330.

**Comment by fredlacis** [comment]
@shixuan-fan Hi there! Is this issue still open? If so, can I work on it? This would be my first issue.

**Comment by shixuan-fan** [comment]
@fredlacis Yes I believe it is still open and I saw you already had a PR to address it ;)
